lms/djangoapps/course_api/blocks/api.py

```diff
@@ -17,7 +17,9 @@ def get_blocks(
         requested_fields=None,
         block_counts=None,
         student_view_data=None,
-        return_type='dict'
+        return_type='dict',
+        block_types=None,
+        lti_url=None
```

Comment:
@nasthagiri Should we use something more like include_fields of proposed Course Discovery API, rather than adding these one add a time? https://openedx.atlassian.net/wiki/display/AC/Course+Discovery+API